cypress/e2e/ResumeParserAPI/pricing.cy.js

```diff
@@ -0,0 +1,123 @@
+import { RESUME_PARSER_API } from '../../resources/data';
+
+describe('Resume Parser API Pricing page', () => {
+    before(() => {
+        cy.visit(RESUME_PARSER_API);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        // Check if the "Accept Cookies" button is visible or exists
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to check "Resume Parser API" title is present', () => {
+        cy.componentVisiblityCheck('h1, h2, h3, h4', 'Resume Parser API');
+    });
+
+    // Test to check visibility of the "Free Plan" title and its price
+    it('3. Test to check "Free Plan" title and price are present', () => {
+        // Verify "Free Plan" title visibility and content
+        cy.get('.card-header .d-block.h3')
```

Comment:
test case is missing to check  
![image](https://github.com/user-attachments/assets/ce0332c6-9c39-437c-ab3b-b81393d489cb)

Reply:
done